src/settings.armlsd

```diff
@@ -32,7 +32,7 @@ reduce heliocorr True           # Perform a heliocentric correction?
 reduce pixelsize 2.5            # The size of the extracted pixels (as an scaled number of Arc FWHM), -1 will not resample
 reduce bgsubtraction perform True       # Subtract the sky background from the data?
 reduce fluxcalibrate True       # Perform a flux calibration
-reduce flexure spec boxcar      # Use boxcar extraction in flexure calculation
+reduce flexure spec boxcar      # Perform flexure correction on spectrum down slit center
```

Comment:
Is this comment still correct? Perhaps give a more general comment with the allowed options?